build/docker/README.md

```diff
@@ -188,7 +188,6 @@ Last updated: October 1, 2017
 | ocaml     |               | 4.05.0        | THRIFT-4517: ocaml 4.02.3 on xenial appears broken |
 | perl      | 5.22.1        | 5.26.1        |       |
 | php       | 7.0.32        | 7.2.19        |       |
-| python    | 2.7.12        | 2.7.15        |       |
 | python3   | 3.5.2         | 3.6.8         |       |
```

Comment:
we probably should update this to 3.7 to the latest version provided by the base docker image (e.g. the ubuntu version we are using)?